### ISSUE TITLE
Coverage difference between bamdst and gatk DepthOfCoverage

### ISSUE THREAD

**Issue by jhylwq123** [shiquan/bamdst#34]
Does bamdst remove duplicates reads automatically? I first used the "time picard MarkDuplicates REMOVE_DUPLICATES=true" function and calculate the coverage using "gatk DepthOfCoverage" based on removed duplicated bam file. The I used bamdst  to calculate the coverage based on unremoved duplicated bam file, because I thought  bamdst has the function of remove duplicates. However, bamdst calculated a much more higher coverage(100X) than gatk. How could this happen?

**Comment by shiquan** [comment]
No, bamdst only read the alignment information from the BAM. The PCR duplicate reads in the report are labeled in the SAM flag. Therefore you should use GATK/samtools/sambamba to mark duplicates before calculating the coverage.
